dnf/cli/commands/module.py

```diff
@@ -40,6 +41,36 @@ def __init__(self, cli):
             super(ModuleCommand.SubCommand, self).__init__(cli)
             self.module_base = dnf.module.module_base.ModuleBase(self.base)
 
+        def _get_modules_from_name_stream_specs(self):
+            modules_from_specs = set()
+            for module_spec in self.opts.module_spec:
+                __, nsvcap = self.module_base._get_modules(module_spec)
+                name = nsvcap.name if nsvcap.name else ""
+                stream = nsvcap.stream if nsvcap.stream else ""
+                if (nsvcap.version and nsvcap.version != -1) or nsvcap.context:
+                    logger.info(_("Only module name, stream or arfitecture is used. "
```

Comment:
typo: ar**f**itecture
And probably and instead of or: `name, stream and architecture are used`

Reply:
@m-blaha Fixed, I added also profile